pkg/provision/providers/qemu/preflight.go

```diff
@@ -75,7 +75,7 @@ func (check *preflightCheckContext) qemuExecutable(ctx context.Context) error {
 }
 
 func (check *preflightCheckContext) checkFlashImages(ctx context.Context) error {
-	for _, flashImage := range check.arch.PFlash(check.options.UEFIEnabled, check.options.ExtraUEFISearchPaths) {
```

Comment:
we should add a check for `swtpm` in the pre-flight checks if secureboot enabled

Reply:
and same with `ovmfctl`